dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsExtendedApi.java

```diff
@@ -263,6 +263,33 @@ public Response aggregatedMetricsPut(@ApiParam(hidden = true) @Parameter(hidden
         return delegate.setAggregatedMetrics(id, versionId, platform, aggregatedMetrics);
     }
 
+    @GET
+    @UnitOfWork(readOnly = true)
+    @RolesAllowed({"curator", "admin", "platformPartner"})
+    @Path("/{id}/versions/{version_id}/aggregatedMetrics")
+    @Produces({MediaType.APPLICATION_JSON})
+    @ApiOperation(value = AggregatedMetricsGet.SUMMARY, notes = AggregatedMetricsGet.DESCRIPTION, authorizations = {
+        @Authorization(value = JWT_SECURITY_DEFINITION_NAME)})
+    @ApiResponses(value = {
+        @ApiResponse(code = HttpStatus.SC_OK, message = AggregatedMetricsGet.OK_RESPONSE, response = Map.class),
```

Comment:
Response should be Metrics.class, not Map.class

Reply:
The annotation `response` still seems off. Maybe not checked in?